setup/update-pib.sh

```diff
@@ -36,47 +44,50 @@ else
 fi
 
 if [ ! -z "$1" ]; then
-	if [ ! "$1" == '-Cerebra' ]; then
+	if [ ! "$1" == '--cerebra' ]; then
 		echo 'This script can only be runned with no parameter, to upgrade cerebra and all Packages'
-		echo 'or with the parameter "-Cerebra" to update only the frontend.'
+		echo 'or with the parameter "--cerebra" to update only the frontend.'
 	        exit 255
 	fi
 fi
 
+# Redirect console output to a log file
+exec > >(tee -a "$LOG_FILE") 2>&1
+
 # Update all packages
 sudo apt-get update
 sudo apt-get -y upgrade
 
-echo -e '\nClean up the html directory...'
-cd $DEFAULT_NGINX_HTML_DIR && sudo -S rm -r *
-cd $USER_HOME
-# Download Cerebra artifact to the working directory
-echo -e '\nDownloading Cerebra application'
-curl $CEREBRA_ARCHIVE_URL_PATH -L --output $ROS_WORKING_DIR/$CEREBRA_ARCHIVE_NAME
-#
-# Unzip cerebra files to nginx
-echo -e '\nUnzip cerebra...'
-#cd $RASP_TMP_FOLDER
+git clone -b "${repo_map[$BACKEND_REPO]}" "$BACKEND_REPO" "$BACKEND_DIR"
+git clone -b "${repo_map[$FRONTEND_REPO]}" "$FRONTEND_REPO" "$FRONTEND_DIR"
```

Comment:
isn't it sufficient to simply do 'git clone "$BACKEND_REPO" "$BACKEND_DIR', since repo_map[...] has value "main" anyway?